src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -270,15 +290,18 @@ static ImmutableHashSet<string> getResourceAssemblyCultureNames(string originalA
         /// When overridden in a derived class, allows substituting an assembly path after we've
         /// identified the context to load an assembly in, but before the assembly is actually
         /// loaded from disk. This is used to substitute out the original path with the shadow-copied version.
-        /// 
-        /// In the case the <param name="assemblyFilePath" /> is moved to a new location then 
-        /// the resource DLLs for the specified <paramref name="resourceAssemblyCultureNames"/> must also be 
-        /// moved _but_ retain their original relative location.
```

Comment:
Does `PrepareSatelliteAssemblyToLoad` still need to ensure the resource DLLs retain their original relative location after moving? If so, consider having similar doc comment at the new method.